script/releases/Env.sol

```diff
@@ -0,0 +1,266 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.12;
+
+import "forge-std/Vm.sol";
+import "zeus-templates/utils/ZEnvHelpers.sol";  
+
+import {TimelockController} from "@openzeppelin/contracts/governance/TimelockController.sol";
+
+/// core/
+import "src/contracts/core/AllocationManager.sol";
+import "src/contracts/core/AVSDirectory.sol";
+import "src/contracts/core/DelegationManager.sol";
+import "src/contracts/core/RewardsCoordinator.sol";
+import "src/contracts/core/StrategyManager.sol";
+
+/// permissions/
+import "src/contracts/permissions/PauserRegistry.sol";
+import "src/contracts/permissions/PermissionController.sol";
+
+/// pods/
+import "src/contracts/pods/EigenPod.sol";
+import "src/contracts/pods/EigenPodManager.sol";
+
+/// strategies/
+import "src/contracts/strategies/EigenStrategy.sol";
+import "src/contracts/strategies/StrategyBase.sol";
+import "src/contracts/strategies/StrategyBaseTVLLimits.sol";
+import "src/contracts/strategies/StrategyFactory.sol";
+
+library Env {
+
+    using ZEnvHelpers for *;
+
+    /// Dummy types and variables to facilitate syntax, e.g: `Env.proxy.delegationManager()`
+    enum DeployedProxy { A }
+    enum DeployedImpl { A }
+    enum DeployedInstance { A }
+
+    DeployedProxy internal constant proxy = DeployedProxy.A;
+    DeployedImpl internal constant impl = DeployedImpl.A;
+    DeployedInstance internal constant instance = DeployedInstance.A;
+
+    /**
+     * env
+     */
+
+    function executorMultisig() internal view returns (address) {
+        return _envAddress("executorMultisig");
+    }
+
+    function opsMultisig() internal view returns (address) {
+        return _envAddress("operationsMultisig");
+    }
+
+    function protocolCouncilMultisig() internal view returns (address) {
+        return _envAddress("protocolCouncilMultisig");
+    }
+
+    function pauserMultisig() internal view returns (address) {
+        return _envAddress("pauserMultisig");
+    }
+
+    function proxyAdmin() internal view returns (address) {
+        return _envAddress("proxyAdmin");
+    }
+
+    function ethPOS() internal view returns (IETHPOSDeposit) {
+        return IETHPOSDeposit(_envAddress("ethPOS"));
+    }
+
+    function timelockController() internal view returns (TimelockController) {
+        return TimelockController(payable(_envAddress("timelockController")));
+    }
+
+    function multiSendCallOnly() internal view returns (address) {
+        return _envAddress("MultiSendCallOnly");   
+    }
+
+    function EIGENPOD_GENESIS_TIME() internal view returns (uint64) {
```

Comment:
This can be removed entirely and also from EigenPod.sol itself

Reply:
I swear we removed this though way back

Reply:
+1, but... it's not removed :D 

I'm gonna keep it here as removing it feels out of scope for this PR.